src/data/un-sdsn-cw/SDSN-CW-EP-LA-2025.json

```
@@ -0,0 +1,61 @@
+  "metric": ["Absolute Emissions", "Capacity", "Generation", "Technology Mix"],
+  "keyFeatures": {
+    "emissionsTrajectory": "Low or no change",
```

```suggestion
    "emissionsTrajectory": "Minor decrease",
```
another missed update